### PR TITLE
change simple_form to form_with

### PR DESCRIPTION
**Pull request by fransan6**
Just in keeping with our other project, I've updated most simple_form to form_with. I also added a validation to the review model for reviews to be between 0 and 5.
Apologies, it incorporates the previous merged PR's edits!

### CODE REVIEW

File: app/controllers/venues_controller.rb
Comment (fransan6) [inline app/controllers/venues_controller.rb:1]: previous PR changes in venues controller so can ignore

File: app/views/venues/index.html.erb
Comment (fransan6) [inline app/views/venues/index.html.erb:1]: previous PR changes in index html so can ignore

File: db/seeds.rb
Comment (fransan6) [inline db/seeds.rb:1]: previous PR changes in seed file so can ignore

**Comment by alexagodzilla** [comment]
LGTM ❤️ 💻 👩‍🚀

**Pull request merged**